atomic_reactor/plugins/exit_remove_worker_metadata.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright (c) 2017 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from atomic_reactor.plugin import ExitPlugin
+from atomic_reactor.constants import PLUGIN_REMOVE_WORKER_METADATA_KEY
+
+__all__ = ('RemoveWorkerMetadataPlugin', )
+
+
+def defer_removal(workflow, cf_map, osbs):
+    key = RemoveWorkerMetadataPlugin.key
+    workflow.plugin_workspace.setdefault(key, {})
```

Comment:
fyi, `setdefault` returns the value. This can be simplified to: 
```python
workspace = workflow.plugin_workspace.setdefault(key, {})
```

Reply:
ok